tools/frrinit.sh.in

```diff
@@ -44,7 +44,7 @@ case "$1" in
 start)
 	daemon_list daemons
 	watchfrr_options="$watchfrr_options $daemons"
-	daemon_start watchfrr
+	daemon_start watchfrr watchfrr
```

Comment:
daemon_start watchfrr all ?

Reply:
nah this is starting watchfrr so we don't want to run vtysh -b on it.

Reply:
I suggest not passing a second argument here, and just change daemon_start to handle a missing second arg as a "not all" case.

Reply:
I think that is a cleaner solution.